.shippable.yml

```diff
@@ -0,0 +1,18 @@
+build_image: freedomjs/freedom
+language: node_js
+node_js:
+- '0.10'
+notifications:
+  email:
+    recipients:
+      - freedom@cs.washington.edu
+    on_success: change
+    on_failure: always
+before_install:
+  - "export DISPLAY=:10.0"
+  - "Xvfb :10 -screen 0 1280x1024x24 &"
+install:
+  - "git pull"
```

Comment:
Sorry i gave you some wrong information here. I just did a quick test on the freedom-for-chrome shippable config and it is correctly retrieving from the current branch. I have a "git branch" and a "git pull" just in case. It will take 0 time and just makes me more comfortable to know that we're up to date on the correct branch. Is that okay to keep?